### ISSUE TITLE
TCP Server unable to Initiate Data Write

### ISSUE THREAD

**Issue by Stuzza21** [jorgen-k/node-red-contrib-tcp-client2#5]
I Am attempting to Have the Server Initiate a Data write to Client on Connection.
Trouble is the Client Data Connection is not Passed on by TCP server on Connection.
When I go to use Action: Write, Payload 'Hello World" there no Topic which contains the IPV6address and Port to send to, and TCP Server Fails.
When the Server Gets an Incoming Message, the Remote Details are contained in the Topic, so the response can be sent. Server just can not initiate. You can see this contained in msg object along with payload.
I have tried using Status Node  to monitor TCP server, but it also does not contain the IPv6 address/port details, so Unable to use Write Command.
Can TCP Server be modified to pass on Client Connection details in Topic when Client  First Connects, as well as the  current incoming client  Client message.
Long Story Short, TCP Server is Unable to Initiate an Outgoing Message, only respond to messages

![TCPservererror](https://github.com/user-attachments/assets/14ad8e26-ca72-4cf7-820c-e38f508cb23c)

**Comment by jorgen-k** [comment]
Fixed. Sorry for the delay.
